### PR TITLE
Allow filtering EC2SecurityGroup by property "GroupName"

### PR DESCRIPTION
**Pull request by leflamm**
* also gitignoring .idea files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /aws-nuke
 /aws-nuke-*
 .glide
+.idea
```
Comment (svenwltr) [inline .gitignore]: We do not want to add any IDE specific ignores here. Every developer should use any IDE or text editor they want and requiring to update our `.gitignore` to reflect the use of all the quirky files some IDEs produce would be a cumbersome tasks.

You rather should [configure a global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: resources/ec2-security-groups.go [hunk 2]
```diff
@@ -81,6 +81,11 @@ func (sg *EC2SecurityGroup) Remove() error {
 	return nil
 }
 
+func (sg *EC2SecurityGroup) Properties() Properties {
+	return NewProperties().
+		Set("GroupName", sg.name)
```
Comment (svenwltr) [inline resources/ec2-security-groups.go]: I suggest to just call it `Name`.

File: .gitignore (hunk 1 above)
Comment (leflamm) [reply .gitignore]: Understood

File: resources/ec2-security-groups.go (hunk 2 above)
Comment (leflamm) [reply resources/ec2-security-groups.go]: Will do

**Comment by leflamm** [comment]
Renamed "GroupName" to "Name", removed Idea-specific entry in `.gitignore`.

**Pull request merged**